src/core/communication.cpp

```diff
@@ -866,17 +866,17 @@ void mpi_sync_topo_part_info_slave(int, int) {
 
 /******************* REQ_BCAST_LBPAR ********************/
 
-void mpi_bcast_lb_params(int field, int value) {
+void mpi_bcast_lb_params(LBParam field, int value) {
 #ifdef LB
-  mpi_call(mpi_bcast_lb_params_slave, field, value);
-  mpi_bcast_lb_params_slave(field, value);
+  mpi_call(mpi_bcast_lb_params_slave, static_cast<int>(field), value);
```

Comment:
why do you cast? you can directly broadcast the enum class

Reply:
an enum class doesn't implicitly cast to an int, therefore we get `cannot convert ‘LBParam’ to ‘int’ for argument ‘2’ to ‘void mpi_call(void (*)(int, int), int, int)’`